webpack.config.js

```diff
@@ -158,7 +161,7 @@ if (config.mode === 'production') {
         '_headers',
         '_redirects',
         // do not cache source maps
-        /.map$/,
+        /\.js\.map$/,
```

Comment:
Oh, I see! I wasn't aware CSS source maps existed. There's no reason to be more strict in this part of the patch, I just wanted to make it consistent with the strictness in the redirect part. And I was being extra cautious in the redirect part so that we still redirect URLs such as `/public/abcd/calltree/?sourceView=something.map`.

Reply:
I will change it all to use .map, for consistency.